llama-index-finetuning/llama_index/finetuning/embeddings/sentence_transformer.py

```diff
@@ -1,91 +1,97 @@
-"""Sentence Transformer Finetuning Engine."""
-
-from typing import Any, Optional
-
-from llama_index.core.base.embeddings.base import BaseEmbedding
-from llama_index.core.embeddings.utils import resolve_embed_model
-from llama_index.finetuning.embeddings.common import (
-    EmbeddingQAFinetuneDataset,
-)
-from llama_index.finetuning.types import BaseEmbeddingFinetuneEngine
-
-
-class SentenceTransformersFinetuneEngine(BaseEmbeddingFinetuneEngine):
```

Comment:
what is the change here for this file?

Reply:
So if a log_path is provided, 
  self.model will be a **TB**SentenceTransformer object
else if no log_path is provided,
  self.model will be a **standard** SentenceTransformer object

Lines 40-43 in sentence_transformer.py

```
if log_path:
            self.model = TBSentenceTransformer(model_id, writer_path = log_path)
        else:
            self.model = SentenceTransformer(model_id)
```